internal/services/mysql/mysql_flexible_server_resource_test.go

```diff
@@ -699,7 +699,7 @@ resource "azurerm_mysql_flexible_server" "test" {
   name                   = "acctest-fs-%d"
   resource_group_name    = azurerm_resource_group.test.name
   location               = azurerm_resource_group.test.location
-  administrator_login    = "adminTerraform"
+  administrator_login    = "_admin_Terraform_892123456789312"
```

Comment:
where does this new value come from? it seems very specific and is a change from the previous config?

Reply:
The new value `_admin_Terraform_892123456789312` comes from the basic func  [here](https://github.com/hashicorp/terraform-provider-azurerm/blob/6293b0379ffe39c6f46e594c06103112f5214595/internal/services/mysql/mysql_flexible_server_resource_test.go#L464). We should not change the value of  ForceNew property administrator_login from `_admin_Terraform_892123456789312` to `adminTerraform` when test case `TestAccMySqlFlexibleServer_updateSku` test sku update.

Reply:
makes sense to me! thanks